packages/studio/frontend/src/views/FlowEditor/inspector/kit/components/blockSelectors/Block/style.scss

```diff
@@ -0,0 +1,82 @@
+@import '~@blueprintjs/core/lib/scss/variables';
```

Comment:
I have seen that the tilde (`~`) when importing from a `node_module` [has been deprecated](https://github.com/webpack-contrib/sass-loader#resolving-import-at-rules). So you can remove it here. Also, since we are using yarn PnP (no `node_module`), I think it's even less important.